R/calcPumpingCosts.R

```diff
@@ -0,0 +1,30 @@
+#' @title calcPumpingCosts
+#' @description provides costs of pumping irrigation water
+#' @cells Only works for magpiecells currently
+#' @irrigation Only works when irrigation = FALSE
+#' @return A magpie object for all magpiecells for all years with information on pumping costs
+#' @author Vartika Singh
+#' #' @seealso \code{\link{readSource}}, \code{\link{calcOutput}}
+#' @importFrom magclass new.magpie
+#' @importFrom utils read.csv2
+#' @examples
+#'
+#' \dontrun{
+#' calcOutput("PumpingCosts")
+#' }
+#'
+
+  calcPumpingCosts <- function() {
+
+    isoCountry  <- toolGetMapping("iso_country.csv")
+    isoCountry1 <- as.vector(isoCountry[, "x"])
+    names(isoCountry1) <- isoCountry[, "X"]
+    x <- new.magpie(cells_and_regions = isoCountry1, years = seq(1995, 2100, by = 5), names = "pumpingcost", fill = 2000)
```

Comment:
is the cost 2000/$?

Reply:
It is 2 cents (0.02 USD per cubic meter) and the water information we have is for million cubic meters, so the cost for a million cubic meters would be 2000 USD. Right?

Reply:
In your equation for water cost in gams, the unit should be in million dollars. Since the unit for water demand is in million m3, the unit of your pumping cost should be in $/m3.